src/Parser/FullParseResult.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace RecursiveTree\Seat\TreeLib\Parser;
+
+use Illuminate\Support\Collection;
+
+class FullParseResult extends ParseResult
```

Comment:
What is this class and what is the advantage over `ParseResult`? If it is supposed to be a general improvement, it should go into `ParseResult` and it should be used by every parser

Reply:
sorry, I somehow missed the PR description